src/util/math.js

```diff
@@ -1,3 +1,21 @@
+const Big = require('big.js')
+
+// Classic native JS sum has floating precision issues:
+// Basic sum: 8.1 + 8.2 = 16.299999999999997
+
+// Transitive issue:
+// 8.3 + 8.7 + 8 + 8.1 = 33.1
+// 8 + 8.7 + 8.1 + 8.3 = 33.099999999999994
```

Comment:
:rofl: 

Thanks for taking care of this. Which tests made you notice?

Reply:
With pagination scenario testing, the cursor pagination rearranged the order differently due to 2-columns order.
And that average check was broken:
https://github.com/stelace/stelace/blob/6b5a6ef5619a71b4f7ca175c46c4883c83708067/test/util.js#L395

I suspected TimescaleDB number precision. It was Javascript in reality...

Reply:
Thanks, what the second/ordering column was?

Reply:
Oh wait, I think I made a mistake. It was the stats check function so there is no second column.
If I remembered correctly, it was the rating test (`avg desc`):
https://github.com/stelace/stelace/blob/6b5a6ef5619a71b4f7ca175c46c4883c83708067/plugins/rating/test/rating.spec.js#L642-L652